tests/security/audit/autrace.pm

```diff
@@ -28,10 +28,6 @@ sub run {
 
     select_console 'root-console';
 
-    # Install related packages and start audit service
-    zypper_call('in audit libaudit1');
-    assert_script_run('systemctl start auditd');
-
     # Use autrace to trace an individual process, output will be logged to audit log
     script_run('autrace /bin/ls /tmp');
     record_info('autrace_output: ', 'autrace will report error here as expected');
```

Comment:
It depends the audit related config file, so please check the return value of `script_run('autrace /bin/ls /tmp');` if failed you can do "record_info", if passed everything is OK.

Reply:
Added record_info for both pass and fail scenarios. It will return error if autrace returns with unexpected reports here.